scripts/west_commands/build.py

```diff
@@ -9,6 +9,11 @@
 import sys
 import yaml
 
+from pathlib import Path
```

Comment:
pathlib is already imported above: https://github.com/nrfconnect/sdk-zephyr/blob/c9eac8821f4cb4554f2ee301472aba704408455b/scripts/west_commands/build.py#L7

Reply:
It does not work without this line